styles.css

```diff
@@ -478,11 +485,209 @@ input {
 footer h4 {
   height: 189px;
   text-align: center;
-  font-family: 'Inter', sans-serif;
+  font-family: "Inter", sans-serif;
   font-style: normal;
   font-weight: bold;
   font-size: 20px;
   line-height: 32px;
   color: #42526e;
   margin-bottom: 147px;
 }
+
+hr {
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
+  .start {
+    background: url("images/Header-llustration-desktop.svg");
+    background-repeat: no-repeat;
+    background-size: 110%;
+    background-position: top -60%;
+    height: 90%;
+  }
+
+  /* nav-bar */
+  .menu {
+    display: none;
+  }
+
+  .navigation {
+    display: inline;
+    width: 100%;
+    font-weight: 700;
+    color: #172b4d;
+  }
+
+  .desktop-nav {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    justify-content: center;
+  }
+
+  .desktop-nav li {
+    margin: 10px;
+  }
+
+  /* banner */
+  .banner {
+    margin-top: 8%;
+    margin-bottom: 0%;
+  }
+
+  .banner h1 {
+    font-size: 40px;
+    width: 690px;
+    margin-left: 5%;
+  }
+
+  .banner p {
+    width: 570px;
+    margin-left: 13%;
+    padding-top: 2%;
+  }
+
+  /* social-media */
+  .social-media ul {
+    position: absolute;
+    left: 50%;
+    right: 50%;
+  }
+
+  /* indicator */
+  .indicator {
+    display: none;
+  }
+
+  /* recent-works */
+  .topper {
+    position: relative;
+    margin-bottom: 0%;
+  }
+
+  .recent-works {
+    margin-top: 3%;
+  }
+
+  .recent-works-cards {
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: row;
```

Comment:
- [ ] Please note that you are required to lay out your work cards using `display: grid;`. The Flexbox box should only be used to lay out the cards` contents. This has been specified in the project requirements.